authors/apps/articles/views.py

```diff
@@ -430,3 +434,204 @@ def get(self, request):
             "favorites": favorite_articles
         }
         return Response(data=favorites, status=status.HTTP_200_OK)
+
+
+class ListReportsAPIView(APIView):
+    """
+    View to handle fetching of reports for particular articles.
+    Allow any users to fetch reports.
+    """
+    serializer_class = ReportsSerializer
+    renderer_classes = (ReportJSONRenderer,)
+    permission_classes = (IsAuthenticated, )
+
+    def get(self, request):
+        """
+        Method to get reports made by current user
+
+        Params
+        -------
+        request: Object with request data and functions.
+
+        Returns
+        --------
+        reports made by current user
+        error message if:
+             - any validation errors encountered
+        """
+        reports = ReportArticles.objects.filter(author=request.user)
+        if reports:
+            serializer = ReportsSerializer(reports, many=True)
+            return Response(serializer.data, status=status.HTTP_200_OK)
+        return Response({"errors" : ERROR_MESSAGES['not found']}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
It shouldn't be a bad request if no reports exist, because there's nothing wrong with the request itself

Reply:
done